src/jit/lower.cpp

```diff
@@ -1499,6 +1499,25 @@ void Lowering::LowerArg(GenTreeCall* call, GenTreePtr* ppArg)
         else
 #endif // !defined(_TARGET_64BIT_)
         {
+
+#ifdef _TARGET_ARM64_
+            // For vararg call, reg args should be all integer.
+            // Insert a copy to reinterpret float as integer.
```

Comment:
Minor suggestion: I would say "Insert a copy to move float value to integer register."